common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -732,6 +733,7 @@ def associate_by_email_if_login_api(auth_entry, backend, details, user, current_
 
     This association is done ONLY if the user entered the pipeline through a LOGIN API.
     """
+    set_custom_attribute('auth_entry', auth_entry)
```

Comment:
Consider the following changes:
1. Potential updates.
```    
    # Enable monitoring of the third-party-auth auth_entry value.
    set_custom_attribute('tpa_pipeline.auth_entry', auth_entry)
```
2. Could you move this so it is always available for third-party-auth?

Reply:
@robrap It will still be available for third-party auth even if we keep it here but I'll move it to the more appropriate place.

Reply:
Done.